source/css/sass/mixins/_typography.scss

```diff
@@ -67,6 +67,25 @@
   vertical-align: middle;
 }
 
+@mixin  text-scripts {
+  font-size: 75%;
+  line-height: 0;
+  position: relative;
+  vertical-align: baseline;
+}
+
+@mixin _font-variant-position($position) {
+  font-size: 75%;
+  line-height: 0;
+  position: relative;
+  vertical-align: baseline;
+
+  @supports (font-variant-position: #{$position}) {
+    position: static;
```

Comment:
Not tested, but do the others need resetting?

Reply:
Good point. Should really be tested with both web fonts and fallback fonts to determine which, if any, properties to be reset.